### ISSUE TITLE
Возможность "отжать" кнопки "Готов ..." (и др.)

### ISSUE THREAD

**Issue by lounres** [m20-sch57/thetruehat#255]
Чтобы, если что, сказать, что ты не готов.

**Comment by alexdany657** [comment]
Есть проблема с отжатием во время того, как сервер уже засек момент, что все готовы и что-то делает. (Выбирает слова, начинает игру и т.д.)

**Comment by lounres** [comment]
> Есть проблема с отжатием во время того, как сервер уже засек момент, что все готовы и что-то делает. (Выбирает слова, начинает игру и т.д.)

Да, но вроде нормальный вариант серверу ругнуться, а клиенту сказать пользователю "так и так, ты тормоз". Единственный аккуратный момент, что это не надо (или, как минимум, не стоит) показывать, как такую же рядовую ошибку; хотя бы уведомление или сообщение "Вы - тормоз".
